libs/libc/stdio/lib_libflushall.c

```diff
@@ -45,6 +45,55 @@
  *
  ****************************************************************************/
 
+int lib_flushall_unlocked(FAR struct streamlist *list)
+{
+  int lasterrno = OK;
+  int ret;
+
+  /* Make sure that there are streams associated with this thread */
+
+  if (list)
+    {
+      FAR FILE *stream;
+      int i;
+
+      /* Process each stream in the thread's stream list */
+
+      for (i = 0; i < 3; i++)
+        {
+          lib_fflush_unlocked(&list->sl_std[i]);
+        }
+
+      stream = list->sl_head;
+      for (; stream != NULL; stream = stream->fs_next)
```

Comment:
Optional, just in case style will not be broken
```suggestion
      for (stream = list->sl_head; stream != NULL; stream = stream->fs_next)
```

Reply:
Done